dali/python/bundle-wheel.sh

```diff
@@ -273,8 +273,7 @@ echo "Fixed rpath of .lib files"
 
 # correct the metadata in the dist-info/WHEEL, e.g.:
 #Root-Is-Purelib: true
-#Tag: cp27-cp27mu-none-manylinux1_x86_64
-sed -i 's/\(Tag:.*\)-none-/\1-/;s/\(Root-Is-Purelib:\) true/\1 false/' ${PKGNAME}-*.dist-info/WHEEL
+sed -i 's/\(Root-Is-Purelib:\) true/\1 false/' ${PKGNAME}-*.dist-info/WHEEL
```

Comment:
Just my curiosity, what `Root-Is-Purelib` is for?

Reply:
It says that the package is pure Python (no compiled extensions). For some reason in our case is set to true, probably because we don't use the setuptools mechanism to build and we build on the side and ship the shared-object library as a binary file. This is saying that it is not a Python-only package

Reply:
Understood. Good to know.